tests/org.jboss.tools.forge.ui.bot.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Require-Bundle: org.jboss.tools.forge.ui;bundle-version="1.0.0",
  org.jboss.tools.forge.core;bundle-version="1.0.0",
  org.jboss.tools.forge.runtime;bundle-version="1.0.0",
- org.jboss.tools.forge.reddeer;bundle-version="[4.4.2,4.5.0]",
+ org.jboss.tools.forge.reddeer;bundle-version="2.2.201",

```

Comment:
since the moved version is lover than the current one. You need to use range properly, otherwise it will still pick cached 4.5.x version